src/generate.js

```diff
@@ -16,12 +16,14 @@ self.generate = function(options) {
 	if (!options.hasOwnProperty('symbols')) options.symbols = false;
 	if (!options.hasOwnProperty('uppercase')) options.uppercase = true;
 	if (!options.hasOwnProperty('excludeSimilarCharacters')) options.excludeSimilarCharacters = false;
+	if (!options.hasOwnProperty('repeatCharactersLimit')) options.repeatCharactersLimit = 0;
 
 	var lowercase = 'abcdefghijklmnopqrstuvwxyz',
 		uppercase = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ',
 		numbers = '0123456789',
 		symbols = '!@#$%^&*()+_-=}{[]|:;"/?.><,`~',
-		similarCharacters = /[ilLI|`oO0]/g;
+		similarCharacters = /[ilLI|`oO0]/g,
+		repeatingCharacters = options.repeatCharactersLimit ? new RegExp('(.)\\1{' + (options.repeatCharactersLimit) + ',}', 'g') : undefined;
```

Comment:
Yes, precisely. You commented earlier about it, if it helps you :)

> By requiring the repeatCharacterLimit option to be at least 2, if this option was provided, we can avoid any serious performance issues. However, it could be left to the user of this package to decide. They should understand the implications if they provided repeatCharacterLimit: 1. IMO, this wouldn't be a likely use case.